staging/src/k8s.io/apiserver/pkg/util/flowcontrol/controller_test.go

```diff
@@ -54,6 +54,14 @@ var mandPLs = func() map[string]*flowcontrol.PriorityLevelConfiguration {
 	return ans
 }()
 
+// intended to be used by the unit tests only.
```

Comment:
This comment is unnecessary, now that this method is impossible to use in anything _but_ unit tests.